defos/src/defos_mac.mm

```diff
@@ -93,6 +93,20 @@ void defos_set_window_title(const char* title_lua) {
     [window setTitle:title];
 }
 
+void defos_set_window_icon(const char *icon_path)
+{
+    NSString *path = [NSString stringWithUTF8String:icon_path];
+    NSImage* image = [[NSImage alloc] initWithContentsOfFile: path];
```

Comment:
This `image` leaks. It needs to be released after giving it to the window button.

Reply:
done